Add a function to switch GA accounts

jhogue opened the pull request:
Instead of two different header files, as both sites use the same one, a new function checks the current multi-site blog ID and spits out a GA tracking code with the proper UA number.

Blog ID numbers taken from the Network Admin Sites Dashboard

jhogue committed:
Add a function to switch GA accounts

Instead of two different header files, as both sites use the same one, a new function checks the current multi-site blog ID and spits out a GA tracking code with the proper UA number.

kangabell commented on functions.php:
```diff
@@ -145,6 +145,28 @@ function ppsri_theme_support() {
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+
+    gtag('config', '".$GA_UA."');
```
this is a nitpick but I like spaces around the `.`'s, so `" . $GA_UA . "` – seems more legible to me. same for line 158 above.

kangabell commented:
Just a little nitpick that I'm fine ignoring!

jhogue committed:
Add spaces around concatenated variables

jhogue committed:
Need an echo statement

Added a message for logged in users as well, just so that we know it is working properly.

jhogue commented:
@kangabell Got it working for sure because I uploaded it all to the live site :grimacing: The two sites are indeed sharing the same Header.php file, and the function now works as expected. No change needed to the PPSDB theme files.
